doc/source/dev/testing.rst

```diff
@@ -0,0 +1,20 @@
+Testing
+=======
+
+The FuseSoC source code contains a large number of tests in the ``tests`` directory.
+These tests use the `pytest framework <https://www.pytest.org>`_.
+
+Running tests
+-------------
+
+.. code-block::
+
+   cd fusesoc/source/directory
+
+   # run all tests
+   pytest
```

Comment:
pytest doesn't work for me anymore and I have to run py.test instead. Not sure if this is just my system or a general thing. Also wonder if we have the same issue as has been found with edalize so that we need to run with `PYTHONPATH=. py.test` to ensure it picks up the local version

Reply:
- py.test is the old binary name, pytest is the newer one (see https://stackoverflow.com/questions/39495429/py-test-vs-pytest-command). `python3 -m pytest` should always work, I've switched to that.
- PYTHONPATH is only needed if you didn't install fuseosc before calling pytest. I've edited the document to explicitly include this step.